src/components/schemas/fields.yaml

```diff
@@ -192,6 +222,63 @@ source:
     Enum value signifying the client from which this expense was created
   example: SLACK
 
+expense_policy_source:
+  type: string
+  enum:
+    - WEBAPP_POLICY_FORM
+    - WEBAPP_POLICY_JSON
+    - WEBAPP_EXPENSE_RECEIPT_AMOUNT_LIMIT
+    - WEBAPP_EXPENSE_MILEAGE_SLAB_RATE
+  description: >
+    Value signifying the client from which this expense policy is created
+  example: WEBAPP_POLICY_FORM
+
+expense_policy_limit_type:
+  type: string
+  enum:
+    - INDIVIDUAL
+    - DAILY
+    - NIGHTLY
+    - WEEKLY
+    - MONTHLY
+    - QUARTERLY
+    - HALF_YEARLY
+    - YEARLY
+  description: >
+    Value signifying the limit type of the expense policy. <br><br>
+    `INDIVIDUAL`: Amount limit will be checked against each expense. <br>
+    `DAILY`: Amount limit will be checked against the total of all expenses in a day. <br>
+    `NIGHTLY`: Amount limit will be checked against the hotel cost per night. <br>
+    `WEEKLY`: Amount limit will be checked against the total of all expenses in a week. <br>
+    `MONTHLY`: Amount limit will be checked against the total of all expenses in a month. <br>
+    `QUARTERLY`: Amount limit will be checked against the total of all expenses in a financial quarter. <br>_Note: expense_fiscal_year_start_month is required when you set this._ <br>
+    `HALF_YEARLY`: Amount limit will be checked against the total of all expenses in a half-year. <br>_Note: expense_fiscal_year_start_month is required when you set this._ <br>
+    `YEARLY`: Amount limit will be checked against the total of all expenses in a financial year. <br>_Note: expense_fiscal_year_start_month is required when you set this._ <br><br>
+    _Note: When this field is set, expense_currency and (expense_amount_max or expense_amount_min) must be set._
```

Comment:
Note: If this field is set(to any value), then the following fields are mandatory in the request body:
1. `expense_currency`
2. either `expense_amount_max` or `expense_amount_min`

Reply:
Note: If this field is set to `QUARTERLY`, `HALF_YEARLY` or `YEARLY`, then the following additional fields are mandatory in the request body:
1. `expense_fiscal_year_start_month`